### PR TITLE
 Laboratory work #1, Pareshina Evgeniya - 20FPL2

### PR DESCRIPTION
**Pull request by eapareshina**

### CODE REVIEW

**Comment by dmitry-uraev** [comment]
@eapareshina,
It's right time to start implementing Lab 1. The deadline for Lab 1 is not that far.

**Comment by dmitry-uraev** [comment]
@eapareshina,
It's right time to start implementing Lab 1. The deadline for Lab 1 is not that far.

File: lab_1/start.py [hunk 1]
```diff
@@ -19,6 +21,8 @@
             file_to_read:
         unknown_text = file_to_read.read()
 
+
```
Comment (dmitry-uraev) [inline lab_1/start.py:24]: waiting for you realisation here

File: lab_1/main.py [hunk 2]
```diff
@@ -6,12 +6,20 @@
...
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
-    pass
+        Splits a text into tokens, converts the tokens into lowercase,
+        removes punctuation and other symbols from words
+        :param text: a text
+        :return: a list of lower-cased tokens without punctuation
+        """
+    if isinstance(text, str):
```
Comment (dmitry-uraev) [inline lab_1/main.py:14]: this check can be rewritten to "if not" statement to simplify functions and avoid unnecessary nesting

**Review by dmitry-uraev — COMMENTED** [review]
@eapareshina ,
Good implementation. Let's try 8 or 10 may be?

1. Remove __pycache__ folder.
2. Try to avoid unnecessary nesting because of "if" conditions. You mau check if conditions at the start of your program and then implement your main logic
3. Be ready to solve tasks and answer my questions on Friday

File: lab_1/main.py (hunk 2 above)
Comment (eapareshina) [reply lab_1/main.py:14]: i've rewritten nearly all parts with 'if' conditions, thank you for the comments!

File: lab_1/start.py (hunk 1 above)
Comment (eapareshina) [reply lab_1/start.py:24]: added

**Pull request closed without merging**